suites/list.go

```diff
@@ -3,24 +3,38 @@
 package suites
 
 import (
+	"fmt"
 	"github.com/dedis/crypto/abstract"
-	"github.com/dedis/crypto/nist"
+	"github.com/dedis/crypto/edwards"
 	"github.com/dedis/crypto/edwards/ed25519"
+	"github.com/dedis/crypto/nist"
 )
 
 // Suites represents a map from ciphersuite name to ciphersuite.
 type Suites map[string]abstract.Suite
 
-func (s Suites) add(suite abstract.Suite) {
-	s[suite.String()] = suite
-}
-
+// Returns a map of all suites
 func All() Suites {
 	s := make(Suites)
+	// The P256-nist implementation doesn't work for the moment
```

Comment:
@ineiti: According to your latest note, P256 does work again and thus the above comment is redundant?